mdn/src/main/java/org/apache/james/mdn/MDNFactory.java

```diff
@@ -28,39 +28,19 @@
  * Class <code>MDNFactory</code> creates MimeMultipartReports containing
  * Message Delivery Notifications as specified by RFC 2298.
  */
-public class MDNFactory
-{
-
-    /**
-     * Default Constructor
-     */
-    private MDNFactory()
-    {
-        super();
-    }
+public class MDNFactory {
     
     /**
      * Answers a MimeMultipartReport containing a
      * Message Delivery Notification as specified by RFC 2298.
      * 
      * @param humanText
-     * @param reporting_UA_name
-     * @param reporting_UA_product
-     * @param original_recipient
-     * @param final_recipient
-     * @param original_message_id
-     * @param disposition
+     * @param mdnReport
      * @return MimeMultipartReport
      * @throws MessagingException
      */
-    static public MimeMultipartReport create(String humanText,
-            String reporting_UA_name,
-            String reporting_UA_product,
-            String original_recipient,
-            String final_recipient,
-            String original_message_id,
-            Disposition disposition) throws MessagingException
-    {
+    public static MimeMultipartReport create(String humanText,
```

Comment:
MDNFactory.create doesn't return an MDN ?

Reply:
I need to introduce a top level MDN object.

And we should be able to do:

```
mdn.asMimeMessage();
```

Reply:
As explained earlier, I will introduce a MDN object.

Reply:
Done